src/components/header/index.jsx

```diff
@@ -236,8 +236,8 @@ const LanguageSelectorMenu = ({ isOpen, toggle, close, ...props }) => {
 					<use href="/assets/header-icons.svg#i18n" />
 				</svg>
 			</button>
-			<nav aria-label="submenu" aria-hidden={'' + !isOpen}>
-				{Object.keys(config.languages).map(id => (
+			<nav aria-label="submenu" aria-hidden={!isOpen}>
+				{typeof window !== 'undefined' && Object.keys(config.languages).map(id => (
```

Comment:
Skip prerendering this as the "currently selected" style is going to be an issue with hydration.

We don't really need to SSR it anyhow, as it's a drop down and therefore hidden by default.